src/Components/sidebar/Sidebar.tsx

```diff
@@ -1,21 +1,22 @@
 import React, { useState } from "react";
-import "./leftside-bar.css";
-import SettingIndex from "../settings-dropdown/setting-index";
+import "./Sidebar.css";
 import { FaMessage } from "react-icons/fa6";
 import { IoSettingsOutline } from "react-icons/io5";
 import { FiLogOut } from "react-icons/fi";
 import { MdNumbers, MdOutlineEmail } from "react-icons/md";
 import { useNavigate } from "react-router-dom";
 import { logoutUser } from "../../services/auth";
-const LeftsideBar: React.FC = () => {
-  const [activeItem, setActiveItem] = useState<string>("settings");
-  const [showSettingIndex, setShowSettingIndex] = useState<boolean>(false);
+import { useAuth } from "../../context/auth-context";
+
+const Sidebar: React.FC = () => {
+  const [activeItem, setActiveItem] = useState<string>("messages");
   const [showSettingsList, setShowSettingsList] = useState<boolean>(false); // New state to control visibility of the settings list
   const Navigate = useNavigate();
+  const { setUser } = useAuth();
 
   const handleCurrentConfigClick = (): void => {
     setActiveItem("settings");
-    Navigate("/setting-index");
+    Navigate("/settings");
   };
 
   const handleSettingsClick = (): void => {
```

Comment:
Navigate("/settings");  // this fixes displaying current configurations when clicking on settings

Reply:
The design team has pushed this to the next sprint so I will resolve this conversation. I think we should keep your idea in mind during the next sprint. It is a good solution.